scripts/update-kubefed-crd.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
would it be simpler if we use something like `go-bindata`, as we've already done in other parts of the codebase, rather than having another shell script to maintain?

Reply:
Yes, I agree with Xavier. I believe that embedding and using the CRD as an asset will be much easier. 
I simply copy-pasted the content of the KubeFedCluster CRD to a constant because I didn't expect that the CRD would change during the time - I kind of considered it as a final CRD. 
In other words, I should have done it by using the `go-bindata` - my bad, sorry.

Reply:
OK. Makes sense.
Moved to go-bindata assets and moved the generation to API - https://github.com/codeready-toolchain/api/pull/57